app/models/flag.rb

```diff
@@ -3,10 +3,8 @@
 require 'avro_turf/messaging'
 
 class Flag
-  include ActiveModel::Serializers::JSON
-  include ActiveModel::Validations
-
-  include RedisConnection
+  include RedisModel
+  include PresentableModel
```

Comment:
A bunch of the common "redis ORM" stuff moved into these mixins. (so could be reused in the new `Rating` model)